sdk/nodejs/Makefile

```diff
@@ -112,6 +112,18 @@ ifneq ($(PULUMI_TEST_COVERAGE_PATH),)
 	if [ -e .nyc_output ]; then yarn run nyc report -r cobertura --report-dir $(PULUMI_TEST_COVERAGE_PATH); fi
 endif
 
+PULUMI_TEST_ORG ?= $(shell pulumi whoami --json | jq ".organizations[0]")
```

Comment:
Hrm I'm surprised this is needed.  Since https://github.com/pulumi/pulumi/pull/17393 all these tests should be running in a local workspace unless `PULUMI_ACCESS_TOKEN` is set.  Which should be faster locally than running the tests against pulumi cloud.  We might have regressed that since that PR, but then we should try to fix that imo.

Reply:
Hmm... when I unset the `PULUMI_ACCESS_TOKEN` in my env, the complaint becomes "no organization named 'organization'". I'm happy to provide more info if you have a suspicion about what's happened?

In any case, I think [the logic says](https://github.com/pulumi/pulumi/blob/9fc89cfb166bd102618f917361f3fc4f6942169a/sdk/nodejs/tests/automation/util.ts#L25) this line would only matter if you had a `PULUMI_ACCESS_TOKEN`, so it shouldn't affect setups without one

Reply:
> I'm happy to provide more info if you have a suspicion about what's happened?

In which test does that happen exactly?

I guess this is fine, but I am slightly more concerned if the tests can't run with `PULUMI_ACCESS_TOKEN` unset, because it would be neat to keep that property (and fix potential regressions of that).  

My other concern would be if you are only logged in locally what happens to `PULUMI_TEST_ORG` then, but it looks like `jq` just returns "null" then, which I guess wouldn't cause much of an issue, since `PULUMI_TEST_ORG` shouldn't be used in that case anyway.

Reply:
Found it: https://github.com/pulumi/pulumi/pull/18754

I'm happy to keep setting `PULUMI_TEST_ORG` here if that's beneficial for your workflow, but I wanted to make sure we can keep running the tests without `PULUMI_ACCESS_TOKEN` set.